src/components/Explorer/Base/Result/MatchingRuns/SerratusApiCalls.jsx

```diff
@@ -2,6 +2,20 @@ import axios from 'axios'
 
 const baseUrl = process.env.REACT_APP_API_URL
 
+export const getAnalysisIndexUrl = (run_id) => {
+    const params = {
+        run_id: run_id,
+    }
+    params[run_id] = run_id
+    return `${baseUrl}/index/run=${run_id}`
+}
+
+export const getAnalysisIndex = async (run_id) => {
+    const AnalysisUrl = getAnalysisIndexUrl(run_id)
```

Comment:
`getAnalysisIndexUrl()` is only used here. suggest merging the functions, dropping unused `params`, and using camelCase:

```javascript
export const getAnalysisIndex = async (run_id) => {
    const analysisUrl = `${baseUrl}/index/run=${run_id}`
    const response = await axios.get(analysisUrl)
    return response.data
}
```